src/domain/collaboration/callout/useCallouts/useCallouts.ts

```diff
@@ -95,6 +95,14 @@ const CALLOUT_DISPLAY_LOCATION_TAGSET_NAME = 'callout-display-location';
 const useCallouts = (params: UseCalloutsParams): UseCalloutsProvided => {
   const journeyTypeName = getJourneyTypeName(params);
 
+  const {
+    canReadCollaboration,
+    canCreateCallout,
+    canCreateCalloutFromTemplate,
+    canReadCallout,
+    loading: authorizationLoading,
+  } = useCollaborationAuthorization();
+
```

Comment:
`canSaveAsTemplate` can be loaded here as well and then we don't need to re-create it on line 149.